.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -376,7 +376,7 @@ jobs:
 {% endblock %}
 {%- endif -%}
 {%- if enable_doc_jobs %}
-  pytorch_doc_build:
+  build-docs:
```

Comment:
Why job name change is needed?
It will break continuity in HUD by creating a new column, wouldn't it?

Reply:
this is a bandaid-like measure to make the job name more sensible and consistent with the others, the HUD should flush out in a couple days but we can group them all together in the meantime, I'll submit a PR there soon